packages/driver/src/cypress/command_queue.ts

```diff
@@ -316,6 +316,11 @@ export class CommandQueue extends Queue<Command> {
       // store the previous timeout
       const prevTimeout = this.timeout()
 
+      // If we have created a timeout but are in an unstable state, clear the timeout in favor of the on load timeout already running.
```

Comment:
```suggestion
      // If we have created a timeout but are in an unstable state, clear the
      // timeout in favor of the on load timeout already running.
```

Just a nitpick, but it's nice not to have to scroll to read the comment 😬

Reply:
I forget not everyone uses ultra-wide monitors 😬